budgetApp1/budgetApp1/TransactionViewController.swift

```diff
@@ -9,6 +9,7 @@ import UIKit
 
 class TransactionViewController: UIViewController {
 
+    let uiSpecs = UISpecs()
```

Comment:
why is this being used again?

Reply:
for the table border colour. Instead of always typing out that long custom colour I just call it from the uiSpecs class I created